### DISCUSSION TITLE
`accept` fails, but all tests are passed?

### DISCUSSION THREAD

**Discussion by bongjunj** [ANLAB-KAIST/KENSv3#93] [Project Q&A 1. Application Layer (Socket)] [answered]
``` c
    int connfd = accept(sfd, /* ... */);
    if (connfd < 0) {
      printf("server: connfd = %d\n", connfd);
      perror("server: accept error");
      return -1;
    }
```

From time to time, `accept` returns -1, but all the tests are passing.
Why such a weird thing happens?

<img width="584" alt="Screen Shot 2022-09-12 at 9 48 19 PM" src="https://user-images.githubusercontent.com/7106939/189657734-f238066d-1faf-41e6-b3b1-9717e7617bcd.png">

**Comment by whkim0** [comment, marked as answer]
It's okay if you pass all the tests. These messages are also okay, as system calls in KENS are cleaned up with a return value of `-1` at  the end of the execution.
